terraform/root_production.tf

```diff
@@ -0,0 +1,125 @@
+module "ecr_jenkins_prod_repository" {
+  source           = "./tdr-terraform-modules/ecr"
+  name             = "jenkins-prod"
+  image_source_url = "https://github.com/nationalarchives/tdr-jenkins/blob/master/docker/Dockerfile-prod"
+  common_tags      = local.common_tags
+  policy_name      = "jenkins_policy"
+  policy_variables = { role_arn = module.jenkins_integration_execution_role.role.arn }
+}
+
+module "jenkins_ecs_prod" {
+  source               = "./tdr-terraform-modules/ecs"
+  common_tags          = local.common_tags
+  project              = "tdr"
+  vpc_id               = module.jenkins_vpc.vpc_id
+  name                 = "jenkins-prod"
+  jenkins              = true
+  task_role_arn        = module.jenkins_prod_ecs_task_role.role.arn
+  execution_role_arn   = module.jenkins_integration_execution_role.role.arn
+  alb_target_group_arn = module.jenkins_alb_prod.alb_target_group_arn
+}
+
+module "jenkins_dns_prod" {
+  source                = "./tdr-terraform-modules/route53"
+  common_tags           = local.common_tags
+  environment_full_name = "management"
+  project               = "tdr"
+  alb_dns_name          = module.jenkins_alb_prod.alb_dns_name
+  alb_zone_id           = module.jenkins_alb_prod.alb_zone_id
+  a_record_name         = "jenkins-prod"
+}
+
+module "jenkins_ec2_prod" {
+  source              = "./tdr-terraform-modules/ec2"
+  ami_id              = module.jenkins_ami.encrypted_ami_id
+  common_tags         = local.common_tags
+  environment         = local.environment
+  name                = "JenkinsProduction"
+  subnet_id           = module.jenkins_vpc.private_subnets[1]
+  security_group_id   = module.jenkins_ec2_security_group.security_group_id
+  attach_policies     = { ec2_policy = module.jenkins_ec2_policy.policy_arn }
+  private_ip          = "10.0.1.222"
+  user_data           = "user_data_jenkins_docker"
+  user_data_variables = { jenkins_cluster_name = "jenkins-prod-${local.environment}" }
```

Comment:
The environment name is mgmt so it's `jenkins-prod-mgmt` The prod part is because it's production Jenkins, not that it's running in the production environment.

Reply:
Ah OK. It's all getting a bit confusing